src/alwaysAck.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+/* @flow */
+
+import type { MessageHandler } from './index'
+
+import { ackOnComplete, ackOnError } from './ackOn'
+import parseAndHandleMessage from './parseAndHandleMessage'
+
+export default (handleError:(e:mixed) => Promise<mixed>, f:(message:Object) => Promise<mixed>): MessageHandler =>
+  parseAndHandleMessage(x => x, ackOnError(handleError), ackOnComplete, f)
```

Comment:
I think I would have done this the other way around.  `alwaysAck` seems more general and reusable than `parseAndHandleMessage`.  That's why I wrote `alwaysAck`: I felt like I didn't have quite the right tool for the job.

Also: this code forces `handleError` to return a promise.  (It also forces `f` to return one, too, but `f` is much more likely to return one, anyways.)